relational_operators/SelectOperator.hpp

```diff
@@ -241,15 +286,18 @@ class SelectWorkOrder : public WorkOrder {
                   std::vector<attribute_id> &&simple_selection,
                   const std::vector<std::unique_ptr<const Scalar>> *selection,
                   InsertDestination *output_destination,
-                  StorageManager *storage_manager)
+                  StorageManager *storage_manager,
+                  const int numa_node = 0)
```

Comment:
@adalbertgerald In the initial distributed version, we don't have NUMA support. So, please remove this for now. Thanks!